pkgs/servers/mastodon/update.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs, stdenv, lib, makeWrapper, yarn2nix, bundix, coreutils,
+  diffutils, nix-prefetch-github, gnused, jq }:
+stdenv.mkDerivation rec {
+  name = "mastodon-update-script";
```

Comment:
Why is this not a shell script with nix-shell?

Reply:
Whatever.

Reply:
😂